CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 All notable changes to the Wazuh app project will be documented in this file.
 
+## Wazuh v4.7.0 - OpenSearch Dashboards 2.7.0 - Revision 00
```

Comment:
**nitpick**: Changelog is pointing to 4.7.0 but the base branch of the pull request is 4.6.0. What version do we want to add?

Reply:
Yes, it was pointing to 4.6.0 because the 4.7.0 branch was not yet created, I have already changed the branch.